src/3box.js

```diff
@@ -17,7 +17,10 @@ const IPFS_OPTIONS = {
   EXPERIMENTAL: {
     pubsub: true
   },
-  preload: { enabled: false }
+  preload: { enabled: false },
+  config: {
+    Bootstrap: [ PINNING_NODE ]
```

Comment:
hmm okay, I was sorta wondering that, but I think it only runs in separate process, so I wonder if you run/use both if there is any performance decrease. Basically we need to pass something here so that it doesn't use the default, mainly 'wss://sgp-1.bootstrap.libp2p.io/' which is throwing errors. Alternatively we could just pass in some other known bootstrap nodes, i.e. 

```
config: {
  Bootstrap: [
    "/dns4/ams-1.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLer265NRgSp2LA3dPaeykiS1J6DifTC88f5uVQKNAd",
    "/dns4/sfo-1.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLju6m7xTh3DuokvT3886QRYqxAzb1kShaanJgW36yx",
    "/dns4/lon-1.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLMeWqB7YGVLJN3pNLQpmmEk35v6wYtsMGLzSr5QBU3",
    "/dns4/sfo-2.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z",
    "/dns4/sfo-3.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLPppuBtQSGwKDZT2M73ULpjvfd3aZ6ha4oFGL1KrGM",
    "/dns4/sgp-1.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLSafTMBsPKadTEgaXctDQVcqN88CNLHXMkTNwMKPnu",
    "/dns4/nyc-1.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLueR4xBeUbY9WZ9xGUUxunbKWcrNFTDAadQJmocnWm",
    "/dns4/nyc-2.bootstrap.libp2p.io/tcp/443/wss/ipfs/QmSoLV4Bbm51jM9C4gDYZQ9Cy3U6aXMJDAbzgu2fzaDs64"
  ]
}

Reply:
If we pass an empty array it will not bootstrap.

Reply:
yeah was just trying that :) that should do it I think